Directory.Build.props

```diff
@@ -33,117 +34,69 @@
     <MicrosoftDataSQLiteVersion>2.0.0</MicrosoftDataSQLiteVersion>
+    <MicrosoftPowerShell5ReferenceAssembliesVersion>1.1.0</MicrosoftPowerShell5ReferenceAssembliesVersion>
     <MicrosoftServiceFabricServicesVersion>2.7.198</MicrosoftServiceFabricServicesVersion>
     <WindowsAzureStorageVersion>8.2.1</WindowsAzureStorageVersion>
 
+    <FSharpCoreVersion>4.2.3</FSharpCoreVersion>
+
+    <SystemManagementAutomationdllVersion>10.0.10586</SystemManagementAutomationdllVersion>
+
     <!-- 3rd party packages -->
     <AWSSDKDynamoDBv2Version>3.3.4.17</AWSSDKDynamoDBv2Version>
     <AWSSDKSQSVersion>3.3.2.7</AWSSDKSQSVersion>
     <BondCoreCSharpVersion>5.3.1</BondCoreCSharpVersion>
     <ConsulVersion>0.7.2.3</ConsulVersion>
+    <GoogleCloudPubSubV1Version>1.0.0-beta13</GoogleCloudPubSubV1Version>
     <GoogleProtobufVersion>3.4.0</GoogleProtobufVersion>
+    <MySqlDataVersion>6.9.9</MySqlDataVersion>
+    <NewRelicAgentApiVersion>5.10.59.0</NewRelicAgentApiVersion>
+    <NewtonsoftJsonVersion>10.0.3</NewtonsoftJsonVersion>
+    <NpgsqlVersion>3.1.9</NpgsqlVersion>
+    <NSubstituteVersion>1.10.0</NSubstituteVersion>
     <ZooKeeperNetExVersion>3.4.9.4</ZooKeeperNetExVersion>
-  </PropertyGroup>
-
-  <PropertyGroup Condition=" '$(BuildFlavor)' == 'Legacy' ">
-    <!-- System packages -->
-    <SystemReflectionMetadataVersion>1.4.2</SystemReflectionMetadataVersion>
-    <SystemValueTupleVersion>4.3.0</SystemValueTupleVersion>
 
-    <!-- Microsoft packages -->
-    <MicrosoftApplicationInsightsVersion>1.2.3</MicrosoftApplicationInsightsVersion>
-    <WindowsAzureServiceBusVersion>4.1.0</WindowsAzureServiceBusVersion>
-    <WindowsAzureStorageVersion>7.0.0</WindowsAzureStorageVersion>
-
-    <!-- 3rd party packages -->
-    <AWSSDKDynamoDBv2Version>3.1.5.2</AWSSDKDynamoDBv2Version>
-    <AWSSDKSQSVersion>3.1.0.9</AWSSDKSQSVersion>
-    <BondCoreCSharpVersion>4.0.1</BondCoreCSharpVersion>
-    <ConsulVersion>0.7.0.3</ConsulVersion>
-    <GoogleProtobufVersion>3.0.0</GoogleProtobufVersion>
-    <ZooKeeperNetExVersion>3.4.8.5</ZooKeeperNetExVersion>
+    <!-- Test related packages -->
+    <BenchmarkDotNetVersion>0.10.9</BenchmarkDotNetVersion>
+    <FluentAssertionsVersion>4.19.4</FluentAssertionsVersion>
+    <XunitSkippableFactVersion>1.3.3</XunitSkippableFactVersion>
+    <xUnitVersion>2.3.0-beta5-build3769</xUnitVersion>
   </PropertyGroup>
 
   <!-- Versioning properties -->
-  <PropertyGroup Condition=" '$(BuildFlavor)' == 'Current' ">
+  <PropertyGroup>
     <VersionPrefix Condition=" '$(VersionPrefix)'=='' ">2.0.0</VersionPrefix>
-    <VersionSuffix Condition=" '$(VersionSuffix)'=='' ">preview3</VersionSuffix>
-  </PropertyGroup>
-
-  <PropertyGroup Condition=" '$(BuildFlavor)' == 'Legacy' ">
-    <VersionPrefix Condition=" '$(VersionPrefix)'=='' ">1.6.0</VersionPrefix>
-    <VersionSuffix Condition=" '$(VersionSuffix)'=='' "></VersionSuffix>
+    <VersionSuffix Condition=" '$(VersionSuffix)'=='' ">preview4</VersionSuffix>
```

Comment:
please don't update this, since we will probably not update the preview number, just the build number (the date), unless we have bigger breaking changes

Reply:
Sure.